ui/app/services/namespace.js

```diff
@@ -20,6 +21,13 @@ export default Service.extend({
 
   inRootNamespace: equal('path', ROOT_NAMESPACE),
 
+  currentNamespacePath: computed('inRootNamespace', 'path', function () {
```

Comment:
When I read "path" I think of the whole path - `currentNamespace` feels a little more intuitive that we're just returning the most nested segment of the path